src/main/java/org/humancellatlas/ingest/core/Checksums.java

```diff
@@ -14,7 +14,7 @@
     @JsonProperty("s3_etag")
     private String s3Etag;
 
-    protected Checksums(String sha1, String sha256, String crc32c, String s3Etag) {
+    public Checksums(String sha1, String sha256, String crc32c, String s3Etag) {
```

Comment:
If you move your `FileServiceTest.java` into the `core` package, then you don't have to make this `public`.